com.fugro.roames.displaz.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "com.fugro.roames.displaz",
+  "branch": "master",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "displaz",
+  "tags": [
+    "master"
+  ],
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "displaz",
+      "buildsystem": "simple",
```

Comment:
Why not `cmake-ninja`?

Reply:
There are more build steps than usual for displaz.